### PR TITLE
Added missing summary description

### PR DESCRIPTION
**Pull request by slobo80**
The summary is missing the important bit of info that is present on the old site at https://msdn.microsoft.com/en-us/library/system.web.httpresponse.setcookie(v=vs.110).aspx

### CODE REVIEW

**Comment by rpetrusha** [comment]
Thanks, @slobo80, for adding the missing internal-only boilerplate. Unfortunately, this is part of a more general problem that in migrating from our existing authoring system to GitHub, all information on internal-only types and members was not migrated.
We'll still merge your PR if you can make one change. The first and second sentences will in the same paragraph, and so the internal-only text will not clearly stand out. Could you add a `<br/>` at the end of the first sentence?

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Change requested in previous comment.

File: xml/System.Web/HttpResponse.xml [hunk 1]
```diff
@@ -2369,7 +2369,7 @@ Response.RedirectToRoutePermanent("Product",
       </Parameters>
       <Docs>
         <param name="cookie">The cookie in the collection to be updated.</param>
-        <summary>Updates an existing cookie in the cookie collection.</summary>
+        <summary>Because the <b>HttpResponse.SetCookie</b> method is intended for internal use only, you should not call it in your code. Instead, you can call the <b>HttpResponse.Cookies.Set</b> method, as the following example shows.<br/>Updates an existing cookie in the cookie collection.</summary>
```
Comment (GuardRex) [inline xml/System.Web/HttpResponse.xml]: Change `HttpResponse.Cookies.Set` to `HttpResponse.Cookies.Add`

Comment (slobo80) [inline xml/System.Web/HttpResponse.xml]: Are you sure? Add is for adding a new cookie whereas Set is for updating the existing cookie in the collection per https://msdn.microsoft.com/en-us/library/system.web.httpcookiecollection.set(v=vs.110).aspx

Comment (GuardRex) [inline xml/System.Web/HttpResponse.xml]: Depends on what @rpetrusha wants to do: The original C# snippet was for `Response.Cookies.Add(MyCookie);`. He called for `Add()` in https://github.com/dotnet/docs/pull/1915#issuecomment-294248842.

Comment (slobo80) [inline xml/System.Web/HttpResponse.xml]: The original was wrong as well. :(

Comment (GuardRex) [inline xml/System.Web/HttpResponse.xml]: I'm inclined to agree with you, but let's hold for a sec until @rpetrusha comes back to make a ruling on it.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the changes, @slobo80!

**Comment by slobo80** [comment]
My pleasure @rpetrusha ! :)

**Pull request merged**